### PR TITLE
363 Bild Statistiken

### LINKED ISSUE #363: Bild-Statistiken
Das PictureGrid sollte für jedes Bild Statistiken anzeigen:
- Anzahl Likes
- Anzahl Kommentare
- (Anzahl Aufrufe)

### PR DESCRIPTION
**Pull request by olschulz**
Closes #363

### CODE REVIEW

File: projects/bp-gallery/src/components/views/bulk-edit/BulkEditView.tsx [hunk 1]
```diff
@@ -119,14 +120,16 @@ const BulkEditView = ({
...
               {(scrollPos: number, scrollHeight: number) => (
-                <PictureScrollGrid
-                  queryParams={getPictureFilters(pictureIds)}
-                  scrollPos={scrollPos}
-                  scrollHeight={scrollHeight}
-                  hashbase={'A'}
-                  showDefaultAdornments={false}
-                  allowClicks={false}
-                />
+                <ShowStats>
```
Comment (MariusDoe) [inline projects/bp-gallery/src/components/views/bulk-edit/BulkEditView.tsx:123]: Not sure we want this here as it is a Curator-only view.

File: projects/bp-gallery/src/components/views/uploads/UploadsView.tsx [hunk 2]
```diff
@@ -53,14 +54,16 @@ const UploadsView = () => {
...
           </Button>
-          <PictureScrollGrid
-            queryParams={{ collections: { id: { null: true } } }}
-            scrollPos={scrollPos}
-            scrollHeight={scrollHeight}
-            hashbase={'uploads'}
-            uploadAreaProps={uploadAreaProps}
-            bulkOperations={[moveToCollection, bulkEdit]}
-          />
+          <ShowStats>
```
Comment (MariusDoe) [inline projects/bp-gallery/src/components/views/uploads/UploadsView.tsx:57]: Not sure either. However, I think one could argue that it feels more natural than in BulkEdit here. Still, I'm not sure we want Stats inside Curator-only views.

**Review by MariusDoe — CHANGES_REQUESTED** [review]
I have annotated views I'm not sure should have the stats. In addition, I think the `LatestPicturesView` should have them.

Regarding styling, I have made some changes on [`363-pfactz-more-styling`](https://github.com/hpi-swa-lab/BP2021RH1/tree/363-pfactz-more-styling). If you like them, you can merge them into here. Otherwise, please tell me why you don't like them ;)

Other than that, nice work!

**Review by MariusDoe — APPROVED** [review]
I like

**Review by DavidSchroschk — APPROVED** [review]
👍 1 💬 "Nice"

**Pull request merged**